tests/e2e/utils/data_builder/metrics.py

```diff
@@ -0,0 +1,130 @@
+from typing import Any, Dict, List, Optional
+
+from mypy_boto3_cloudwatch.type_defs import DimensionTypeDef, MetricDataQueryTypeDef
+
+from aws_lambda_powertools.metrics import MetricUnit
+from tests.e2e.utils.data_builder.common import build_random_value
+
+
+def build_metric_query_data(
+    namespace: str,
+    metric_name: str,
+    period: int = 60,
+    stat: str = "Sum",
+    dimensions: Optional[List[DimensionTypeDef]] = None,
+) -> List[MetricDataQueryTypeDef]:
+    dimensions = dimensions or []
```

Comment:
Should we add a docstring here the same way you do in other methods? I guess we want to add it by default to every method?

Reply:
Good call! I'm adding to every "public" method known to us.